CONTRIBUTING.md

```diff
@@ -0,0 +1,371 @@
+# Website
+
+> Hack for LA's website: https://www.hackforla.org
```

Comment:
Notice this line has hackforla URL. This file needs to be customised for 100 Automations. Please make the changes and update your PR.